ppa/archive/gale.py

```diff
@@ -160,23 +160,57 @@ def _make_request(
             requests.codes.unauthorized,
             requests.codes.server_error,
         ]:
+            # Extract item ID from the URL (format: .../v1/item/GALE%7C{item_id})
+            item_id = None
+            if "GALE%7C" in rqst_url:
+                item_id = rqst_url.split("GALE%7C")[-1]
+
             # occasionally we get a 500 error when indexing all pages
             # refreshing API key and trying again, but log the error
             if resp.status_code == requests.codes.server_error:
-                # Not sure yet if response has any meaningful content
-                logger.error(f"500 error on {rqst_url}: {resp.content}")
+                # Log concise error info (status, URL, item ID)
+                logger.error(
+                    f"500 server error on {rqst_url}, Item ID: {item_id or 'N/A'}"
+                )
 
             # If we get a 401 or 500 on a request that requires an api key,
-            # get a fresh key and then try the same request again
-            if requires_api_key and retry < 1:
+            # get a fresh key and then try the same request again (up to 3 retry attempts)
+            if requires_api_key and retry < 3:
```

Comment:
I had this set to retry once because when I initially implemented, that was all that was needed - when the API key expires, you get an error, and then when you refresh it and it works. I don't know what the behavior is with the 500 error or if it's consistent enough for us to test.

That said, I like this addition. Suggest moving it to a class variable, something like `max_retries` so it's easy to find and change if we want to adjust.